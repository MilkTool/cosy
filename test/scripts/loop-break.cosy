for i in 0..10 {
    print i
    if i > 4 break
    print 'still ' + i
}
print 'all done!'
