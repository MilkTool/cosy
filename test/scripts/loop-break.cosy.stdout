0
still 0
1
still 1
2
still 2
3
still 3
4
still 4
5
all done!
